common/borders/borders-en.html

```diff
@@ -0,0 +1,77 @@
+---
+{
+	"title": "Borders",
+	"language": "en",
+	"altLangPage": "borders-fr.html",
+	"breadcrumbs":
+	[
+		{
+			"title": "GCWeb home",
+			"link": "https://wet-boew.github.io/themes-dist/GCWeb/index-en.html"
+		}
+	],
+	"secondlevel": false,
+	"dateModified": "2023-06-20",
+	"share": "true"
+}
+---
+
+<h3>CSS Class</h3>
```

Comment:
This create an issue with the page structure when jekyl generate the HTML final page. The heading level 2 is missing. 
Heading should always be in sequence.

Considering this page is short, I suggest to simply use a paragraph here.

```suggestion
<p>CSS Class</p>
```